samples/getting-started/azure-quantum/Getting started with Qiskit and IonQ on Azure Quantum.ipynb

```diff
@@ -0,0 +1,652 @@
+      "text/plain": [
+       "<Figure size 504x360 with 1 Axes>"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Submit the circuit to run on Azure Quantum\n",
+    "qpu_job = qpu_backend.run(circuit, shots=1024)\n",
+    "job_id = qpu_job.id()\n",
+    "print(\"Job id\", job_id)\n",
+    "\n",
+    "# Monitor job progress and wait until complete:\n",
+    "job_monitor(qpu_job)\n",
+    "\n",
+    "# Get the job results (this method also waits for the Job to complete):\n",
+    "result = qpu_job.result()\n",
+    "print(result)\n",
+    "counts = {format(n, \"03b\"): 0 for n in range(8)}\n",
+    "counts.update(result.get_counts(circuit))\n",
+    "print(counts)\n",
+    "plot_histogram(counts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82b0b1d8",
+   "metadata": {},
+   "source": [
+    "# Quantum Phase Estimation (QPE)\n",
+    "\n",
+    "The `azure-quantum[qiskit]` also supports more advanced examples. You can take any existing Qiskit code sample and by changing only a few lines of code, submit it to Azure Quantum.\n",
+    "\n",
+    "Let's use the QPE example from Qiskit textbook: https://qiskit.org/textbook/ch-algorithms/quantum-phase-estimation.html"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ec9beeb2",
+   "metadata": {},
+   "source": [
+    "Below, all cells from the example notebook are copy-pasted. Feel free to give it a try yourself!"
```

Comment:
We may need a third-party notice for this usage. You should be able to check by adding a cgmanifest.json file to the repo indicating the source for this copy–paste.

Reply:
Let's take out this sample completely, I don't think there is much added value anyway.

Reply:
Fair enough! Not arguing for that, though, just so much as making sure we dot our eyes and cross our tees on code reuse.